### PR TITLE
README: Remove note about not supporting EAB.

### PR DESCRIPTION
**Pull request by cpu**
Since https://github.com/letsencrypt/pebble/pull/288 Pebble **does** support external account binding (EAB). This commit updates the "Limitations" section of the README accordingly.

### CODE REVIEW

**Review by jsha — APPROVED** [review]
Sweet, thanks!

**Comment by cpu** [comment]
@jsha Could you merge this when you have a chance? Ty!

**Comment by felixfontein** [comment]
@cpu good to see you're still around :)

**Comment by cpu** [comment]
@felixfontein Regretfully just for drive-by README updates and not anything more interesting! :laughing:

**Comment by jsha** [comment]
Thanks for the reminder! I was waiting for CI, and forgot to circle back around 🤦🏻

**Review by AlphaMale1st — COMMENTED** [review]
My apologies my children, 4give me. Wipe the sleep out, wake up theres fresh aroma brewing all around.

**Pull request merged**
